contract/src/main/resources/swagger/kafbat-ui-api.yaml

```diff
@@ -4693,6 +4713,10 @@ components:
                               type: string
                             password:
                               type: string
+                            keystoreType:
```

Comment:
@iamtakingiteasy openapi spec here is outdated/a backup, typespec (*.tsp) files need to be obe updated to

Reply:
@Haarolean It is updated accordingly as well, see `model KeystoreConfig` / `enum StoreType` / `enum SecurityProtocol` and corresponding references. I don't see anything missing; `kafkaConnect.keystoreType` is updated to `StoreType` among other things.